pkg/webhook/elasticquota/quota_handler.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package elasticquota
+
+import (
+	"k8s.io/klog/v2"
+
+	"github.com/koordinator-sh/koordinator/apis/extension"
+	"github.com/koordinator-sh/koordinator/pkg/scheduler/plugins/elasticquota/core"
+)
+
+// webhook works with multiple copies at the same time, so it needs to watch other copies' writes to task effect locally.
+// OnQuotaAdd/Update/DeleteHandlers no need to check, and if its parent doesn't exist, just create.
+
+func (qt *quotaTopology) OnQuotaAdd(obj interface{}) {
+	quota := toElasticQuota(obj)
+	if quota == nil {
+		klog.Errorf("cannot convert to *ElasticQuota:%v", obj)
+		return
+	}
+
+	quotaInfo := core.NewQuotaInfoFromQuota(quota)
+	qt.lock.Lock()
+	defer qt.lock.Unlock()
+
+	qt.quotaInfoMap[quotaInfo.Name] = quotaInfo
+	if _, ok := qt.quotaHierarchyInfo[quotaInfo.Name]; !ok {
```

Comment:
if _, ok := qt.quotaHierarchyInfo[quotaInfo.Name]; !ok {} and if qt.quotaHierarchyInfo[quotaInfo.ParentName] == nil {}  , use same style, I prefer "== nil"

Reply:
fixed